docs/Events/bpmn-events/import-done.md

```diff
@@ -4,9 +4,42 @@ title: import.done
 nav_order: 4
 description: import.done event
 permalink: /events/bpmn-events/import-done
-last_modified_date: 2020-04-27T17:54:08+0000
+last_modified_date: 2020-09-21T12:00:00+0000
 parent: "BPMN Events"
 grand_parent: "Events"
 ---
 
 # import.done
+
+This event is fired by [importXML API](/general/api-reference/bpmn-js/importXML) with data containing `errors` and `warnings` after the import ends. This allows users to listen to whether the import of a diagram was successful or not.
```

Comment:
Maybe this phrase would be better?: `This allows users to check if the import was successful or not`